web/compatibility.js

```diff
@@ -14,7 +14,9 @@
  */
 /* globals VBArray, PDFJS */
 
-'use strict';
+(function pdfjsWrapper() {
+  // Use strict in our context only - users might not want it
```

Comment:
Can you remove (unimportant?) comment and rename `pdfjsWrapper`=>`compatibilityWrapper`? Looks good after that.